File: app/src/main/java/com/example/proyecto/domain/Util/New/Pruebas/GetQuotesUseCase.java
package com.example.proyecto.domain.Util.New.Pruebas;

import com.example.proyecto.data.model.Unit;

import java.util.List;

public class GetQuotesUseCase {

    private QuoteRepository repository = new QuoteRepository();

    public List<Unit> invoke(){
        return repository.getAllQuotes();
    }
}

File: app/src/main/java/com/example/proyecto/domain/GetQuotesUseCase.kt
package com.example.proyecto.domain

import com.example.proyecto.data.QuoteRepository
import com.example.proyecto.data.model.QuoteModelUnidadSesion

class GetQuotesUseCase {

    private val repository = QuoteRepository()

    suspend operator fun invoke(codigo:String):List<QuoteModelUnidadSesion>{
        return repository.getAllQuotes(codigo)
    }
}